test-runner/src/main/scala/org/typesafe/spark/mesos/framework/runners/MesosIntegrationTestRunner.scala

```diff
@@ -17,9 +17,8 @@ object MesosIntegrationTestRunner {
     printMsg("TestResults:")
     result.foreach(printMsg)
 
-    //TODO: Generate some JUnit style report for CI
-    if (result.exists(_.startsWith("TestFailed"))) {
-      System.exit(1)
+    if (result.exists(_.contains("FAILED"))) {
+      printMsg(Console.RED + "ERROR: One or more tests failed" + Console.RESET)
```

Comment:
I am not sure I understand. The System.exit(1) actually never exited the sbt process. It throws exception for me.